### DISCUSSION TITLE
Unable to run split preconditioning

### DISCUSSION THREAD

**Discussion by amosaha** [idaholab/moose#21670] [Q&A General] [answered]
![image](https://user-images.githubusercontent.com/93416273/180583156-0b368a59-b1f4-4d49-a0dd-18c707e5f92a.png)
When using the example provided by the official website of the split module, the problem in the figure appears. How should I solve it?
![image](https://user-images.githubusercontent.com/93416273/180583244-a3b39a6f-6d18-4d03-aa64-37a88a9b92a7.png)

**Comment by GiudGiud** [comment, marked as answer]
Hello

The [Splits] syntax isnt supported syntax outside of phase_field I think, but field splits are still definitely possible.
Rename this block as Preconditioning.

Guillaume
